lms/djangoapps/course_blocks/management/commands/tests/test_generate_course_blocks.py

```diff
@@ -44,6 +44,21 @@ def test_generate_all(self):
         self._assert_courses_not_in_block_cache(self.course_1.id, self.course_2.id)
         self.command.handle(all=True)
         self._assert_courses_in_block_cache(self.course_1.id, self.course_2.id)
+        with patch(
+            'openedx.core.lib.block_structure.factory.BlockStructureFactory.create_from_modulestore'
+        ) as mock_update_from_store:
+            self.command.handle(all=True)
+            mock_update_from_store.assert_not_called()
+
+    def test_generate_force(self):
+        self._assert_courses_not_in_block_cache(self.course_1.id, self.course_2.id)
+        self.command.handle(all=True)
+        self._assert_courses_in_block_cache(self.course_1.id, self.course_2.id)
+        with patch(
+            'openedx.core.lib.block_structure.factory.BlockStructureFactory.create_from_modulestore'
+        ) as mock_update_from_store:
```

Comment:
Nit: Dedupe: extract this patch to a self.patched_update in setUp, and change the two invocations to `with self.patched_update as mock_update_from_store:`.